### PR TITLE
[minor] Support option to skip cluster's pre-checks during pipeline execution

### PR DESCRIPTION
**Pull request by andrercm**
This PR adds the argument `--skip-pre-check` that allows skipping the pre-check tasks that are executed when pipelines are started.

This has been implemented in:

- mas install
- mas update
- mas upgrade

**mas update:**

image

This also works with FVTs (ansible-fvt)

**mas install (using fvt-personal):**

image

ansible-fvt pull request: https://github.ibm.com/maximoappsuite/ansible-fvt/pull/398

### CODE REVIEW

File: tekton/src/pipelines/taskdefs/cluster-setup/ocp-verify-all.yml.j2 [hunk 1]
```diff
@@ -3,6 +3,12 @@
     kind: Task
     name: mas-devops-ocp-verify-all
   params:
-    {{ lookup('template', 'taskdefs/common/cli-params.yml.j2') | indent(4) }}
```
Comment (whitfiea) [inline tekton/src/pipelines/taskdefs/cluster-setup/ocp-verify-all.yml.j2:6]: don't we still need taskdefs/common/cli-params.yml.j2' as it is still in the task?

File: tekton/src/pipelines/taskdefs/cluster-setup/ocp-verify.yml.j2 [hunk 2]
```diff
@@ -3,10 +3,11 @@
     kind: Task
     name: mas-devops-ocp-verify
   params:
-    {{ lookup('template', 'taskdefs/common/cli-params.yml.j2') | indent(4) }}
```
Comment (whitfiea) [inline tekton/src/pipelines/taskdefs/cluster-setup/ocp-verify.yml.j2:6]: don't we still need taskdefs/common/cli-params.yml.j2' as it is still in the task?

**Review by whitfiea — CHANGES_REQUESTED** [review]

File: tekton/src/pipelines/taskdefs/cluster-setup/ocp-verify.yml.j2 (hunk 2 above)
Comment (andrercm) [reply tekton/src/pipelines/taskdefs/cluster-setup/ocp-verify.yml.j2:6]: oh oh... yeah thanks for catching that, i'll have it fixed.

File: tekton/src/pipelines/taskdefs/cluster-setup/ocp-verify-all.yml.j2 (hunk 1 above)
Comment (andrercm) [reply tekton/src/pipelines/taskdefs/cluster-setup/ocp-verify-all.yml.j2:6]: oh oh... yeah thanks for catching that, i'll have it fixed.

**Review by whitfiea — APPROVED** [review]

**Pull request merged**